### DISCUSSION TITLE
Is it possible to delete a document with all its subcollections?

### DISCUSSION THREAD

**Discussion by piotrw91** [prescottprue/cypress-firebase#293] [General]
Hello there!

I've been using cypress-firebase recently, and there is one thing I wonder about.

If I run the bellow call as CLI command, I manage to successfully delete the document and also all it's subcollections.

`firebase firestore:delete -r /path/document --yes` 

But If I try to achieve the same using the library, as below, it is not possible.  It only deletes the document, but all subcollections are still accessible. 
```
cy.callFirestore('delete', `/path/document`, {
    recursive: true,
  });
```

Is it possible to delete a document with all its subcollections?

**Comment by piotrw91** [comment]
@prescottprue any ideas if this is doable? Sorry for the poke ;)

**Comment by prescottprue** [comment]
@piotrw91 Currently this is not a feature available through the `firebase-admin` SDK which `cypress-firebase` uses under the hood to interact with Firebase (plus a little bit with `firebase` for client side logic in `cy.login`). That said, I don't see why we couldn't support falling back to internally calling firebase-tools (the cli) to make this a feature in the future

To do this currently with firebase-tools, you should be able to call it through [`cy.exec`](https://docs.cypress.io/api/commands/exec#Syntax).
